src/app/containers/create-movie/MovieForm.jsx

```diff
@@ -0,0 +1,159 @@
+              type="text"
+              name="description"
+              placeholder="Description"
+            />
+            {errors.description && touched.description && (
+              <div className="text-red-500 text-m mt-1">
+                {errors.description}
+              </div>
+            )}
+
+            <Field
+              className="w-96 px-3 py-2 border border-white-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+              as="select"
+              name="genres.id"
+              onChange={(e) => {
+                const selectedGenre = genres.find(
+                  (genre) => genre.id.toString() === e.target.value,
+                );
+                setFieldValue(
+                  "genres.id",
+                  selectedGenre ? selectedGenre.id : "",
+                );
+                setFieldValue(
+                  "genres.name",
+                  selectedGenre ? selectedGenre.name : "",
+                );
+              }}
+            >
+              <option value="">Select Genre</option>
+              {genres.map((genre) => (
+                <option key={genre.id} value={genre.id}>
+                  {genre.name}
+                </option>
+              ))}
+            </Field>
+            {errors.genres?.id && touched.genres?.id && (
+              <div className="text-red-500 text-m mt-1">{errors.genres.id}</div>
+            )}
+            {errors.genres?.name && touched.genres?.name && (
+              <div className="text-red-500 text-m mt-1">
+                {errors.genres.name}
+              </div>
+            )}
+
+            <Field
+              className="w-96 px-3 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+              type="text"
+              name="videoSource"
+              placeholder="Video Source"
+            />
+            {errors.videoSource && touched.videoSource && (
+              <div className="text-red-500 text-m mt-1">
+                {errors.videoSource}
+              </div>
+            )}
+
+            <Field
+              className="w-96 px-3 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+              type="text"
+              name="cast.role"
+              placeholder="Cast Role"
+            />
+            {errors.cast?.role && touched.cast?.role && (
+              <div className="text-red-500 text-m mt-1">{errors.cast.role}</div>
+            )}
+
+            <Field
+              className="w-96 px-3 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+              type="text"
+              name="cast.crew.id"
+              placeholder="Crew ID"
+            />
+            {errors.cast?.crew?.id && touched.cast?.crew?.id && (
+              <div className="text-red-500 text-m mt-1">
+                {errors.cast.crew.id}
+              </div>
+            )}
+
+            <Field
```

Comment:
Since all these fields are repetitive, we could extract into a separate component that could be reused